css/editable-js.css

```diff
@@ -89,10 +89,6 @@
         align-items: center;
     }
 
-    .run {
-        margin-right: 10px;
-    }
-
     .output {
```

Comment:
I think because of this change, we are losing the gap between the buttons and the output in the JS examples:

<img width="273" alt="screen shot 2018-03-01 at 4 59 33 pm" src="https://user-images.githubusercontent.com/432915/36878037-0240a936-1d72-11e8-96d4-80621835fe22.png">

cf

<img width="279" alt="screen shot 2018-03-01 at 4 59 59 pm" src="https://user-images.githubusercontent.com/432915/36878039-05c10a1a-1d72-11e8-8f06-5b819f8ff58d.png">